nixos/tests/all-tests.nix

```diff
@@ -266,6 +266,7 @@ in {
   dovecot = handleTest ./dovecot.nix {};
   drawterm = discoverTests (import ./drawterm.nix);
   drbd = handleTest ./drbd.nix {};
+  drbd-driver = handleTest ./drbd-driver.nix {};
```

Comment:
The `drbd` test above uses the `services.drbd.enable` option, which already enables the kernel module. This test is redundant.

https://github.com/NixOS/nixpkgs/blob/44072e24566c5bcc0b7aa9178a0104f4cfffab19/nixos/modules/services/network-filesystems/drbd.nix#L43

Reply:
The [drbd](https://github.com/birkb/nixpkgs/blob/drbd-kernel-module/nixos/tests/drbd.nix) test validates the userland utilities, but not the kernel driver. The in-tree driver is loaded to create the DRBD devices, but no special driver checks are done.

The [drbd-driver](https://github.com/birkb/nixpkgs/blob/drbd-kernel-module/nixos/tests/drbd-driver.nix) test loads the out-of-tree driver (provided by this package) and validates the version of the loaded driver.

I think we need both tests...